package com.desafio.cwi.exceptions;

public class CpfNotFoundException  extends RuntimeException {
	public CpfNotFoundException(String msg) {
		super(msg);
	}
}
